services/util/loopy.py

```diff
@@ -19,43 +21,34 @@ def __init__(self, query_url, params, page_size=100, order_by='_id ASC'):
         self.result_count = self.get_count()
         self.total_pages = int(math.ceil(float(self.result_count)/float(page_size)))
 
-    def get_query_string(self, filter_prefix='filter'):
-        query_string = '?'
+    def get_query_string(self, limit, skip, order):
+        and_dict = {"and": []}
 
         for param in self.params:
             for case in switch(param['query_type']):
                 if case('between'):
-                    layout = '{}[where][{}][{}][0]={}&{}[where][{}][{}][1]={}&'
-                    query_string += layout.format(
-                        filter_prefix,
-                        param['property_name'],
-                        param['query_type'],
-                        param['query_value'][0],
-                        filter_prefix,
-                        param['property_name'],
-                        param['query_type'],
-                        param['query_value'][1])
+                    between_dict = {param['property_name']: {"between": param['query_value']}}
+                    and_dict['and'].append(between_dict)
                     break
                 if case('where'):
-                    query_string += '{}[{}][{}]={}&'.format(
-                        filter_prefix,
-                        param['query_type'],
-                        param['property_name'],
-                        param['query_value'])
+                    where_dict = {param['property_name']: param['query_value']}
+                    and_dict['and'].append(where_dict)
                     break
                 if case('neq'):
-                    query_string += '{}[where][{}][neq]={}&'.format(
-                        filter_prefix,
-                        param['property_name'],
-                        param['query_value'])
+                    between_dict = {param['property_name']: {"neq": param['query_value']}}
+                    and_dict['and'].append(between_dict)
                     break
                 if case():
                     # default, could also just omit condition or 'if True'
                     print('huh?')
-        return query_string
+        where_dict = {'where': and_dict, 'limit': limit, 'skip': skip, 'order':order}
```

Comment:
do we want 'order' to be an arg to this method or in the constructor. might be confusing in 2 places.